### DISCUSSION TITLE
Is there a way to rescue an error?

### DISCUSSION THREAD

**Discussion by tobiasz-p** [AutomaApp/automa#1279] [General]
Is there a way to rescue an error? E.g. My workflow sometimes fails so I want to refresh page when html element is not found.

**Comment by SuperTempAccount** [comment]
I haven't tried, but could possibly be done with the conditions block. Do a check for if the DOM element is present and reload the page if not. 
https://docs.automa.site/blocks/conditions.html

Otherwise, maybe a Javascript block if you need to do some more complex checks.

Edit: 
1. Click the gear icon to see the block settings. 
2. Go to the `On error` tab and enable it. 
3. Select `Execute fallback` from the dropdown. 
4. You can then link the fallback to a block that reloads the tab.
